### ISSUE TITLE
Cannot enable debug logs!

### ISSUE THREAD

**Issue by Kate-T** [anbox/anbox#849]
1. Please check that no similar bug is already reported. Have a look on the list of open bugs at https://github.com/anbox/anbox/issues
2. Make sure you are running the latest version of Anbox before reporting an issue. Update snap to latest: `snap refresh --devmode --edge anbox`
3. Make sure you have debug logs enabled:
`sudo snap set anbox debug.enable=true`
4. Reproduce the error while debug logs enabled.
5. Run the anbox logs collection utility and attach the tar file.
`sudo /snap/bin/anbox.collect-bug-info `

6. ** Please paste the result of `anbox system-info` below:**
```
[anbox system-info
version: 4
snap-revision: 92
os:
  name: Ubuntu
  version: 16.04.5 LTS (Xenial Xerus)
  snap-based: true
kernel:
  version: Linux version 4.4.0-131-generic (buildd@lgw01-amd64-015) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.10) ) #157-Ubuntu SMP Thu Jul 12 15:51:36 UTC 2018
  binder: true
  ashmem: true
graphics:
  egl:
    vendor: Mesa Project
    version: 1.4 (DRI2)
    extensions:
      - EGL_CHROMIUM_sync_control
      - EGL_EXT_create_context_robustness
      - EGL_EXT_image_dma_buf_import
      - EGL_EXT_image_dma_buf_import_modifiers
      - EGL_KHR_config_attribs
      - EGL_KHR_create_context
      - EGL_KHR_create_context_no_error
      - EGL_KHR_fence_sync
      - EGL_KHR_get_all_proc_addresses
      - EGL_KHR_gl_renderbuffer_image
      - EGL_KHR_gl_texture_2D_image
      - EGL_KHR_gl_texture_3D_image
      - EGL_KHR_gl_texture_cubemap_image
      - EGL_KHR_image
      - EGL_KHR_image_base
      - EGL_KHR_image_pixmap
      - EGL_KHR_no_config_context
      - EGL_KHR_reusable_sync
      - EGL_KHR_surfaceless_context
      - EGL_KHR_wait_sync
      - EGL_MESA_configless_context
      - EGL_MESA_drm_image
      - EGL_MESA_image_dma_buf_export
      - EGL_NOK_swap_region
      - EGL_NOK_texture_from_pixmap
      - EGL_NV_post_sub_buffer
      - EGL_WL_bind_wayland_display
  gles2:
    vendor: Intel Open Source Technology Center
    vendor: OpenGL ES 3.0 Mesa 17.2.8
    extensions:
      - GL_EXT_blend_minmax
      - GL_EXT_multi_draw_arrays
      - GL_EXT_texture_filter_anisotropic
      - GL_EXT_texture_compression_dxt1
      - GL_EXT_texture_format_BGRA8888
      - GL_OES_compressed_ETC1_RGB8_texture
      - GL_OES_depth24
      - GL_OES_element_index_uint
      - GL_OES_fbo_render_mipmap
      - GL_OES_mapbuffer
      - GL_OES_rgb8_rgba8
      - GL_OES_standard_derivatives
      - GL_OES_stencil8
      - GL_OES_texture_3D
      - GL_OES_texture_float
      - GL_OES_texture_float_linear
      - GL_OES_texture_half_float
      - GL_OES_texture_half_float_linear
      - GL_OES_texture_npot
      - GL_OES_vertex_half_float
      - GL_EXT_texture_sRGB_decode
      - GL_OES_EGL_image
      - GL_OES_depth_texture
      - GL_OES_packed_depth_stencil
      - GL_EXT_texture_type_2_10_10_10_REV
      - GL_OES_get_program_binary
      - GL_APPLE_texture_max_level
      - GL_EXT_discard_framebuffer
      - GL_EXT_read_format_bgra
      - GL_EXT_frag_depth
      - GL_NV_fbo_color_attachments
      - GL_OES_EGL_image_external
      - GL_OES_EGL_sync
      - GL_OES_vertex_array_object
      - GL_ANGLE_texture_compression_dxt3
      - GL_ANGLE_texture_compression_dxt5
      - GL_EXT_robustness
      - GL_EXT_texture_rg
      - GL_EXT_unpack_subimage
      - GL_NV_draw_buffers
      - GL_NV_read_buffer
      - GL_NV_read_depth
      - GL_NV_read_depth_stencil
      - GL_NV_read_stencil
      - GL_EXT_draw_buffers
      - GL_EXT_map_buffer_range
      - GL_KHR_debug
      - GL_KHR_robustness
      - GL_OES_depth_texture_cube_map
      - GL_OES_surfaceless_context
      - GL_EXT_color_buffer_float
      - GL_EXT_separate_shader_objects
      - GL_EXT_shader_integer_mix
      - GL_INTEL_performance_query
      - GL_EXT_compressed_ETC1_RGB8_sub_texture
      - GL_EXT_draw_buffers_indexed
      - GL_EXT_draw_elements_base_vertex
      - GL_EXT_polygon_offset_clamp
      - GL_EXT_texture_border_clamp
      - GL_KHR_blend_equation_advanced
      - GL_KHR_context_flush_control
      - GL_KHR_robust_buffer_access_behavior
      - GL_OES_draw_buffers_indexed
      - GL_OES_draw_elements_base_vertex
      - GL_OES_sample_shading
      - GL_OES_sample_variables
      - GL_OES_shader_multisample_interpolation
      - GL_OES_texture_border_clamp
      - GL_OES_texture_stencil8
      - GL_EXT_blend_func_extended
      - GL_KHR_no_error
      - GL_EXT_clip_cull_distance
      - GL_MESA_shader_integer_functions
]
```

**Please describe your problem:**
I can't enable debug logs (in order to post my problem here), because when I enter `sudo snap set anbox debug.enable=true` I get the following error message:
'error: cannot perform the following tasks:
- Run configure hook of "anbox" snap (snap "anbox" has no "configure" hook)'

**What were you expecting?:**
I was expecting to enable debug logs so that I could perform all the steps listed above, in order to sensibly post here about my actual bug.

**Additional info:**
I checked to see if there had been a similar problem concerning enabling debug logs, but could find nothing.

Thanks in advance!

**Comment by morphis** [comment]
I guess you're on the beta channel of the Anbox snap which does not yet have the changes to enable debug logging. If you switch to edge you will have them. Try `snap refresh --edge --devmode anbox`

**Comment by Kate-T** [comment]
Thanks, Morphis!

When I tried that command, I got this error message:
'error: cannot perform the following tasks:
- Download snap "anbox" (134) from channel "edge" (received an unexpected http response code (503) when trying to download https://fastly.cdn.snapcraft.io/download-origin/fastly/Nr9K6UJaIOD8wHpDEQl16nabFFt9LLEQ_134.snap?token=1533668400_8af08f4b4cee101d9d5bb155157ce7b0c39a5681)'

Does this mean I have to uninstall everything and start over from scratch?

**Comment by morphis** [comment]
No. The error means snapd failed to download the snap due to some connection issue. Just try it again.Am 07.08.2018 17:56 schrieb Kate-T <notifications@github.com>:Thanks, Morphis!
When I tried that command, I got this error message:
'error: cannot perform the following tasks:
Download snap "anbox" (134) from channel "edge" (received an unexpected http response code (503) when trying to download https://fastly.cdn.snapcraft.io/download-origin/fastly/Nr9K6UJaIOD8wHpDEQl16nabFFt9LLEQ_134.snap?token=1533668400_8af08f4b4cee101d9d5bb155157ce7b0c39a5681)'
Does this mean I have to uninstall everything and start over from scratch?
